docs/whatsnew/7.0.rst

```diff
@@ -288,6 +289,29 @@ New ``Lorentz2D`` model
 A new 2D Lorentzian model has been added to the ``astropy.modeling``
 package.
 
+.. _whatsnew-7.0-masked-coordinates:
+
+Support for masks in ``coordinates``
+====================================
+
+Initial support for masks has been implemented for |SkyCoord|, as well as for
+coordinate frames and coordinate representations.
+
+Like more generally for |Masked| instances, one can access the mask of using
+``.mask``, check whether the underlying data are masked using ``masked``,
+access underlying unmasked data using ``.unmasked``, and get a new instance
+with masked data replaced with a fill value using the ``filled()`` method.
```

Comment:
Why aren't there any tests for this?

Reply:
Because it just uses `__setitem__`... But fair enough, one should at least have one test!

Reply:
Done